### ISSUE TITLE
Adjust URL column for Environments Table

### ISSUE THREAD

**Issue by RudyOnRails** [department-of-veterans-affairs/va.gov-team#37462]
### Description
Platform suggested I open this issue to make an improvement. The current table at https://depo-platform-documentation.scrollhelp.site/developer-docs/Environments.603488279.html#:~:text=%C2%A0-,https%3A//staging.va.gov,-Staging lists the URL's for the frontend, but sandbox-api has no FE in reality, but has staging.va.gov listed.

I suggest either: 

- the URL column for sandbox should be empty, or
- the URL’s in the URL column should just be the domains for the actual api instead of the FE URL.

for example:
| Environment | Description | URL                |
|-------------|-------------|--------------------|
| Development | xxx         | dev-api.va.gov     |
| Sandbox     | xxx         | sandbox-api.va.gov |
| Staging     | xxx         | staging-api.va.gov |
| Production  | xxx         | api.va.gov         |

### Relevant URLs
- https://dsva.slack.com/archives/CBU0KDSB1/p1645575411311169

**Comment by gmaciolek** [comment]
Hi Kevin, Thanks for your suggestion! Tagging @LindseySaari for here expertise here. Once we're all settled on what changes to be made, I'm happy to update the page.

**Comment by gmaciolek** [comment]
I checked in with Lindsay and it seems like the sandbox URL on the page was a copy/paste mistake. I've updated the table with the accurate URL: https://depo-platform-documentation.scrollhelp.site/developer-docs/Environments.603488279.html

Does that change address your concerns?

**Comment by RudyOnRails** [comment]
@gmaciolek not quite, `https://sandbox.va.gov/` does not exist.

I think this makes more sense:



Environment | Description | vets-api URL
-- | -- | --
Development | xxx | dev-api.va.gov
Sandbox | xxx | sandbox-api.va.gov
Staging | xxx | staging-api.va.gov
Production | xxx | api.va.gov

**Comment by gmaciolek** [comment]
Yes, thank you! Lindsey corrected herself right after I made the update when I was in a meeting. Your suggested table layout is great. I will update the page again.
